Server/src/models/entities/Datapoints.ts

```diff
@@ -0,0 +1,43 @@
+import {Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn, JoinColumn, ManyToOne} from "typeorm";
+import { Dataset } from "./Dataset";
+import { Units } from "./Units";
+
+
+/**
+ * The entity annotation indicates that a table is being created
+ */
+@Entity()
+export class Datapoints {
+
+    @PrimaryGeneratedColumn()
+    id: number
+
+    @Column()
+    datasetId: number
+
+    @Column()
+    name: string
+
+    @ManyToOne(type => Dataset)

```

Comment:
can you explain me how this works? Dont these 3 line mean that Datapoints will also store the dataset?